### ISSUE TITLE
Auth using Github identity

### ISSUE THREAD

**Issue by apenwarr** [tailscale/tailscale#446]
Many people would like to use Github as the auth layer rather than GSuite, Office365, etc. This is not hard to add, but until #413 is available, it probably won't do what people are actually hoping for, which is letting you invite other github users to share devices with you. So we might delay it until after #413.

<img src="https://frontapp.com/assets/img/favicons/favicon-32x32.png" height="16" width="16" alt="Front logo" /> [Front conversations](https://app.frontapp.com/open/top_2aqxd)

**Comment by cocoonkid** [comment]
yes, please!

**Comment by ramyfication** [comment]
Added :)
